app/konnect/dev-portal/customization/custom-domain.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Add a Custom Domain
+no_version: true
+---
+
+All Dev Portals have an assigned, generated Dev Portal URL. To further customize your Dev Portal, you can add a custom domain. When set up properly, users can access both the assigned URL and the custom URL.
+
+## Prerequisites
+
+* Access to your organization's {{site.konnect_short_name}} Admin UI.
+* A domain with access to configure that domain's CNAME. Kong does not offer any custom domains as a service.
+* A signed certificate, which comes with a private key.
+* Your organization's FQDN (Full Qualified Domain Name), which is listed in the {{site.konnect_short_name}} Admin UI. For example, `https://kong121212.portal.konnect.konghq.com/`.
+
+## Direct your CNAME to the Dev Portal assigned URL
+
+In your custom domain DNS records, direct your CNAME to your Dev Portal's assigned URL, or FQDN. Check your DNS resolution to ensure this was done properly.
+
+## Add a custom Dev Portal domain
+
+Add a custom Dev Portal domain through your organization's {{site.konnect_short_name}} Admin UI.
+
+1. Click **Dev Portal**, then **Portal URL**.
+
+1. Enter the following fields.
+
+   * **Custom Portal URL**: Enter the full domain, including subdomain (if applicable). Don't include a path. It's not necessary to include the URL protocol, for example, `https://`.
+
+   * **X.509 Certificate**: Obtain your certificate and add it to the field, including the certification header and footer. For example, a header may look like: `-----BEGIN CERTIFICATE-----`.
+
+        This certificate ensures that your custom domain site is trusted and that the browser doesn't show a warning when users land on the page.
+
+   * **Private Key**: Add the private key that came with your X.509 certificate.
+
+   {:.note}
+   > **Note**: You will **not** be able to see view or edit keys in the UI once you submit the form. To change your custom domain URL, X.509 certificate, or private key, delete your certificate and re-enter your custom domain URL, X.509 certificate, and private key.
+
+1. Once you've entered all fields, click **Save Certificate**.
+
+You'll see the custom domain listed in your Dev Portal under your assigned Dev Portal URL, even if it doesn't work properly. Always test to see if your custom domain works.
```

Comment:
Can you add an "official" step to verify the URL? A final "verify that your URL works by visiting it". Seems like common sense but many people don't read the stuff that appears after the steps.

Reply:
Yes, will add!